### ISSUE TITLE
Automatic polys (poly that can build without player's presence not only rebuild) 

### ISSUE THREAD

**Issue by ScolarIronFist** [Anuken/Mindustry-Suggestions#2627]
**Describe the content or mechanics you are proposing.**

Polys being able to automatically build (They automatically go out to rebuild structures but they dont go to automatically build structures unless the player is building them)

**Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.**

Normally in the game when you have a lot of polys and you can build lots of things you place your schematics and buildings yet you build them one at the time. even if you build really fast you are building only one at the time and also only the ones that are within your building radius.

Having polys building automatically without having to need the player near would increase by a LOT the maneuverability of players by being able to build everything they need at the same time instead of being building structures one by one. 

The polys would split in small groups of 2 or even 3 to build the structures depending on the distance and how much resources they need. They would build the nearest buildings ordered and from the nearest they would build the cheap buildings first and then the expensive buildings.

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
2. - [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
3. - [X] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by Yell245** [comment]
Uhhhhhhh... Maybe you should rephrase it to something like "multithread poly building"?... Also, you'll still build everything at the same, but if resources end, then you'll need to wait for *all* buildings to be built. What's better: two built turrets or 10 unbuilt ones?

**Comment by IIINSTINCT** [comment]
This was suggested before... well duplicate suggestions is allowed now but it was suggested before and never implemented welp to bad it wont happen

**Comment by ScolarIronFist** [comment]
> This was suggested before... well duplicate suggestions is allowed now but it was suggested before and never implemented welp to bad it wont happen



> Uhhhhhhh... Maybe you should rephrase it to something like "multithread poly building"?... Also, you'll still build everything at the same, but if resources end, then you'll need to wait for _all_ buildings to be built. What's better: two built turrets or 10 unbuilt ones?

I am the one that suggested the payload cannon and unit transfer between sectors for V7. 
Unlike you....
Is amazing how pesky are peasants these days.

**Comment by itcannotbe** [comment]
Sorry to break it to you, unit transfers between sectors have been planned for a long time already.

**Comment by itcannotbe** [comment]
Sorry to break it to you, unit transfers between sectors have been planned for a long time already.

Also you're not the first guy to suggest sending units from one sector to another. 1320 was earlier than you.

Also there's this which predate your suggestion.
![Screenshot_20210628_125111](https://user-images.githubusercontent.com/45280919/123582181-f915da00-d80f-11eb-832e-c1013a7885c8.jpg)

**Comment by oinkymoo** [comment]
> I am the one that suggested the payload cannon and unit transfer between sectors for V7.

your point being?

**Comment by Kyllingene** [comment]
Ah yes, because obviously you were the only one to think of payload cannons. What an arrogant thing to say, you aristocrat.
